llvm/lib/Target/X86/X86ISelLowering.h

```diff
@@ -1673,7 +1673,8 @@ namespace llvm {
     SDValue LowerTRUNCATE(SDValue Op, SelectionDAG &DAG) const;
     SDValue LowerFP_TO_INT(SDValue Op, SelectionDAG &DAG) const;
     SDValue LowerFP_TO_INT_SAT(SDValue Op, SelectionDAG &DAG) const;
-    SDValue LowerLRINT_LLRINT(SDValue Op, SelectionDAG &DAG) const;
+    SDValue LowerLRINT_LLRINT(SDValue Op, const X86Subtarget &STI,
```

Comment:
Do you need to provide the X86Subtarget arg? Subtarget is available in the X86TargetLowering class

Reply:
Good catch! Done.